src/i18n/es/ui.ts

```diff
@@ -1,113 +1,122 @@
-	// Footer
+	// Pie de página
 	'footer.privacyPolicy': 'Política de privacidad',
-	// `<ThemeToggleButton>` acessibility labels
+	// Etiquetas de accesibilidad para <ThemeToggleButton>
 	'themeToggle.useLight': 'Usar tema claro',
 	'themeToggle.useDark': 'Usar tema oscuro',
-	// Used in previous/next page links at the bottom of pages
+	// Se utiliza en los enlaces de página anterior/siguiente en la parte inferior de las páginas
 	'articleNav.nextPage': 'Siguiente página',
-	'articleNav.prevPage': 'Página anterior',
-	// Used in `<Since>`: Added in: v0.24.0 [NEW]
+	'articleNav.prevPage': 'Atrás',
+	// Se utiliza en <Since>: Agregado en: v0.24.0 [NUEVO]
 	'since.addedIn': 'Agregado en:',
 	'since.new': 'Nuevo',
 	'since.beta': 'Beta',
-	// Installation Guide
-	'install.autoTab': 'CLI Automático',
-	'install.manualTab': 'Instalación Manual',
-	// `<DeployGuidesNav>` vocabulary
+	// Guía de instalación
+	'install.autoTab': 'CLI automática',
+	'install.manualTab': 'Configuración manual',
+	// Vocabulario de <DeployGuidesNav>
 	'deploy.sectionTitle': 'Guías de implementación',
-	'deploy.altSectionTitle': 'Más guías de despliegue',
-	'deploy.filterLabel': 'Filtrar por tipo de despliegue',
+	'deploy.altSectionTitle': 'Más guías de implementación',
+	'deploy.filterLabel': 'Filtrar por tipo de implementación',
 	'deploy.ssrTag': 'SSR',
 	'deploy.staticTag': 'Estático',
-	// CMS Guides vocabulary
+	// Vocabulario de guías de CMS
 	'cms.navTitle': 'Más guías de CMS',
-	// Migration Guides vocabulary
+	// Vocabulario de guías de migración
 	'migration.navTitle': 'Más guías de migración',
-	// Recipes vocabulary
+	// Vocabulario de recetas
 	'recipes.navTitle': 'Más recetas',
-	// `<ContributorList>` fallback text
-	'contributors.seeAll': 'Ver a todos los contribuidores',
-	// Fallback content notice shown when a page is not yet translated
+	// Vocabulario de <RecipeLinks>
+	'recipesLink.singular': 'Receta relacionada:',
+	'recipesLink.plural': 'Recetas relacionadas',
+	// Texto de respaldo para <ContributorList>
+	'contributors.seeAll': 'Ver todos los colaboradores',
+	// Aviso de contenido de respaldo que se muestra cuando una página aún no está traducida
 	'fallbackContent.notice':
-		'Esta página aún no está disponible en tu idioma, mientras tanto verás la versión en inglés. ¡Ayúdanos a traducirla!',
-	'fallbackContent.linkText': 'Más información sobre cómo contribuir',
-	// 404 Page
+		'Esta página aún no está disponible en tu idioma, así que te mostramos la versión en inglés. ¡Puedes ayudar traduciéndola!',
+	'fallbackContent.linkText': 'Obtén más información sobre cómo puedes contribuir',
+	// Página 404
 	'404.title': 'No encontrado',
 	'404.content': 'Esta página no está en nuestro sistema solar.',
-	'404.linkText': 'Volver a la página principal.',
-	// `<CodeSnippet>` vocabulary
-	'codeSnippet.terminalCaption': 'Ventana del terminal',
-	// `<LanguageSelect>` vocabulary
+	'404.linkText': 'Llévame a casa.',
```

Comment:
yo lo dejaroa como estaba originalmente
```suggestion
	'404.linkText': Volver a la página principal.',
```